src/coreclr/vm/wasm/calldescrworkerwasm.cpp

```diff
@@ -0,0 +1,16 @@
+#include <interpexec.h>
```

Comment:
nit missing license header.

Also, you can apply this change (forward declaration) to fix build issue https://github.com/dotnet/runtime/pull/116353/files#diff-3fa3e7e639c020a2e3a925aec85ffc58bf260b487b1f6a38798812ac66300d36